packages/breadboard/package.json

```diff
@@ -110,6 +120,23 @@
       "output": [
         "docs/api/"
       ]
+    },
+    "test:node": {
+      "dependencies": [
+        "test:node:js",
+        "test:node:ts"
+      ]
+    },
+    "test:node:js": {
+      "dependencies": [
+        "build:tsc"
+      ],
+      "command": "node --test dist/**/*.node.test.js",
+      "files":[],
+      "output": []
+    },
+    "test:node:ts": {
+      "command": "node --test --loader ts-node/esm tests/**/*.node.test.ts"
```

Comment:
I don't think we should use ts-node. tsc + node seems simpler.

Reply:
changed over in https://github.com/breadboard-ai/breadboard/pull/642/files#r1483545313